dbms/src/Flash/Planner/PhysicalPlan.cpp

```diff
@@ -275,11 +275,11 @@ void PhysicalPlan::buildBlockInputStream(DAGPipeline & pipeline, Context & conte
     root_node->buildBlockInputStream(pipeline, context, max_streams);
 }
 
-PipelinePtr PhysicalPlan::toPipeline()
+PipelinePtr PhysicalPlan::toPipeline(PipelineExecutorStatus & exec_status, Context & context)
 {
     assert(root_node);
     PipelineBuilder builder{log->identifier()};
-    root_node->buildPipeline(builder);
+    root_node->buildPipeline(builder, context, exec_status);
```

Comment:
> Maybe we can define a helper struct to contain buildPipeline parameters? such as:
> 
> ```
> struct buildPipelineParams {
>   PipelineExecutorStatus exec_status;
>   Context * context;
> };
> ```

Not necessary? Since so many place can use struct as Param, IMHO only if the api is not stable and need to change frequently, we can use struct to hold all the param